packages/node/src/events/propose-install-virtual/controller.ts

```diff
@@ -34,10 +34,24 @@ export default async function proposeInstallVirtualEventController(
     return;
   }
 
-  const relayedProposalMsg: ProposeVirtualMessage = {
-    from: requestHandler.publicIdentifier,
-    type: NODE_EVENTS.PROPOSE_INSTALL_VIRTUAL,
-    data: nodeMsg.data
+  const relayedProposalMsg: NodeOperation = {
+    meta: {
+      from: requestHandler.publicIdentifier,
+      requestId: ""
```

Comment:
Probably outside the scope of this PR but we should discuss needing `requestId` - from what I've seen so far it's been empty / not referenced at all.